etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml

```diff
@@ -0,0 +1,140 @@
+dataset:
+  update_period_days: 365
+  sources: []
+definitions:
+  common:
+    sources: []
+    origins:
+      - producer: World Bank World Development Indicators
+        title: UNICEF, World Health Organization and World Bank
```

Comment:
If multiple sources are being listed, we should probably separate them using `;`